pallets/domains/src/lib.rs

```diff
@@ -208,116 +224,58 @@ pub mod pallet {
             origin: OriginFor<T>,
             target: <T::Lookup as StaticLookup>::Source,
             full_domain: DomainName<T>,
-            content: Content,
             expires_in: T::BlockNumber,
         ) -> DispatchResult {
             ensure_root(origin)?;
             let owner = T::Lookup::lookup(target)?;
 
-            Self::do_register_domain(owner, full_domain, content, expires_in, IsForced::Yes)
+            Self::do_register_domain(owner, full_domain, expires_in, IsForced::Yes)
         }
 
-        /// Sets the domain inner_value to be one of Subsocial account, space, or post.
+        /// Change the record associated with a domain name.
+        ///
+        /// ## Record Deposit
```

Comment:
Is it Okay to have H2 headings (two hashes in Markdown) in a doc for a function? Maybe this is better?
```suggestion
        /// **Record Deposit:**
```